variables.tf

```diff
@@ -88,3 +88,41 @@ variable "values" {
   default     = ""
   description = "Additional yaml encoded values which will be passed to the Helm chart, see https://hub.helm.sh/charts/stable/cluster-autoscaler"
 }
+
+variable "argo_namespace" {
+  type        = string
+  default     = "argo"
+  description = "Namespace to deploy ArgoCD application CRD to"
+}
+
+
+variable "argocd_application" {
```

Comment:
variable naming. Everywhere else is only "argo_smth" isnt it better to keep this naming? For this variable it means: "argo_application"